### PR TITLE
chore(22652): Rename hook file to TS when appropriate

### PR DESCRIPTION
**Pull request by vanch3d**
https://hivemq.kanbanize.com/ctrl_board/57/cards/22652/details/

Changed the file type of all relevant hook files to TS. 

While not changing much of the code, it allows these files to be considered for code coverage (since the unit tests exclude TSX files)

Also added quite a few tests 

And updated MSW and Vite (fix #451 )

### CODE REVIEW

File: hivemq-edge/src/frontend/src/api/hooks/useConnection/useGetAdaptersStatus.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { MemoryRouter } from 'react-router-dom'
+
+import { server } from '@/__test-utils__/msw/mockServer.ts'
+import { AuthProvider } from '@/modules/Auth/AuthProvider.tsx'
+
+import { handlers } from './__handlers__'
+import { useGetAdaptersStatus } from '@/api/hooks/useConnection/useGetAdaptersStatus.ts'
+
+const wrapper: React.JSXElementConstructor<{ children: React.ReactElement }> = ({ children }) => (
```
Comment (h2xd) [inline hivemq-edge/src/frontend/src/api/hooks/useConnection/useGetAdaptersStatus.spec.tsx:12]: question(non-blocking): potential wrapper that can be exported into test utils

File: hivemq-edge/src/frontend/src/api/hooks/useConnection/useGetBridgesStatus.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { MemoryRouter } from 'react-router-dom'
+
+import { server } from '@/__test-utils__/msw/mockServer.ts'
+import { AuthProvider } from '@/modules/Auth/AuthProvider.tsx'
+
+import { handlers } from './__handlers__'
+import { useGetBridgesStatus } from '@/api/hooks/useConnection/useGetBridgesStatus.ts'
+
+const wrapper: React.JSXElementConstructor<{ children: React.ReactElement }> = ({ children }) => (
```
Comment (h2xd) [inline hivemq-edge/src/frontend/src/api/hooks/useConnection/useGetBridgesStatus.spec.tsx:12]: question: okay, yeah you might want to export and reimport it here :D

**Review by h2xd — APPROVED** [review]
LGTM! 🐊 
_I've some glassy eyes from clicking viewed_

File: hivemq-edge/src/frontend/src/api/hooks/useConnection/useGetBridgesStatus.spec.tsx (hunk 2 above)
Comment (vanch3d) [reply hivemq-edge/src/frontend/src/api/hooks/useConnection/useGetBridgesStatus.spec.tsx:12]: There are only 27 of them :-) 
Fixing it

**Pull request merged**